packages/contracts/contracts/utils/UUPSProxy.sol

```diff
@@ -3,15 +3,19 @@
 pragma solidity 0.8.10;
 
 import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import "./AppStorage.sol";
 
 /// @title UUPSProxy
-/// @author Aragon Association - 2022
+/// @author Giorgi Lagidze - Aragon Association - 2022
 /// @notice This contract is only meant to be used for deploy purposes.
-/// TODO: Need to be fully tested, and decide to use the second address param as admin address.
-contract UUPSProxy is ERC1967Proxy {
+contract UUPSProxy is ERC1967Proxy, AppStorage {
```

Comment:
Adding Natspec would also be helpful, even if it is for auditors

Reply:
DONE.